### PR TITLE
Ok/054 build test

### PR DESCRIPTION
**Pull request by OldBeardedCoder**
Contains SafeUnsocket, BiggerGoldPiles and NoFlawGems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,8 +162,14 @@ Thanks!
 
 ## Upcoming Release
 * Added ElementalSkills option (+Cold,Lightning,Magic,Poison Skills)
```
Comment (tlentz) [inline README.md:164]: do you want to include this in the release? is there a UI option for it?

File: VERSION.txt [hunk 2]
```diff
@@ -1 +1 @@
-v0.5.3
+v0.5.4-build-test.2
```
Comment (tlentz) [inline VERSION.txt:1]: i'll change this at release time to v0.5.4

File: cfg.json [hunk 3]
```diff
@@ -1,11 +1,13 @@
 {
-    "Version": "v0.5.3",
+    "Version": "v0.5.4",
     "SourceDir": "",
     "OutputDir": "",
     "MeleeSplash": true,
     "IncreaseStackSizes": true,
     "IncreaseMonsterDensity": 1,
     "EnableTownSkills": true,
+    "BiggerGoldPiles": true,
```
Comment (tlentz) [inline cfg.json:9]: should all these options be false by default?

**Review by tlentz — COMMENTED** [review]
some comments

File: cfg.json (hunk 3 above)
Comment (OldBeardedCoder) [reply cfg.json:9]: Definitely.

File: README.md (hunk 1 above)
Comment (OldBeardedCoder) [reply README.md:164]: There is now.

File: cfg.json (hunk 3 above)
Comment (OldBeardedCoder) [reply cfg.json]: Options updated to all off except startswithcube

File: README.md (hunk 1 above)
Comment (tlentz) [reply README.md:164]: this can be added to v0.5.4 in the readme

**Review by tlentz — APPROVED** [review]

**Pull request merged**